source/dem/dem.cc

```diff
@@ -51,6 +51,13 @@ DEMSolver<dim>::DEMSolver(DEMSolverParameters<dim> dem_parameters)
   , repartition_frequency(parameters.model_parameters.repartition_frequency)
   , insertion_frequency(parameters.insertion_info.insertion_frequency)
   , physical_properties(parameters.physical_properties)
+  , rayleigh_time_step(
+      M_PI_2 * parameters.physical_properties.diameter *
+      sqrt(2 * parameters.physical_properties.density *
+           (2 + parameters.physical_properties.poisson_ratio_particle) *
+           (1 - parameters.physical_properties.poisson_ratio_particle) /
+           parameters.physical_properties.youngs_modulus_particle) /
+      (0.1631 * parameters.physical_properties.poisson_ratio_particle + 0.8766))
```

Comment:
The calculation of the rayleigh time-step should be move to a small function.
Eventually we might want to calculate multiple critical time-step (hertz, etc). So instead of hardcoding it in the constructor, it would be better to move that to a small function and to call that function in the constructor.